### PR TITLE
TurtleShroom Productions X

### PR DESCRIPTION
**Pull request by TurtleShroom**

### CODE REVIEW

**Comment by loup99** [comment]
1. You could have suggested the Zalmoxism gods directly to me rather than including them here: that is regardless how they will be implemented as they are beyond the scope of this.
2. I will ignore the non-Spartan material for the purpose of the review and merge, given that they are not a part of the focus this pull-request. We don't want any more pull-requests changing hundreds of files or any debate on matters which do not pertain Sparta delaying this. 
3. If possible, I need public domain links for all non-original sound and art which is included here. Otherwise I will have to find them myself, and if I can't they naturally won't be included. 
4. Make usage of existant files rather than creating new ones for every single line of code, you can create sub-categories within those.

File: WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+					c_monemvasia = {
+						culture = spartan
+						gender_succ = true_cognatic
+					}
+					c_monemvasia = { 
+						set_name = Sparta
+						set_coa = k_sparta
+					}
+					b_sparta = {
+						set_coa = k_sparta
```
Comment (loup99) [inline WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt]: I'm not sure this works given the way barony CoAs are handled.

File: WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+								religion = hellenic_pagan
+								religion = cybelian_pagan
+								religion = neoplatonist
+								religion = mithraic_pagan
+								religion = mastimani
+							}
+						}
+					}
+				}
+			}
```
Comment (loup99) [inline WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt]: Redundant with above.

File: WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+					trait = eunuch
+				}
+				create_random_soldier = { # Slave
+					random_traits = yes
+					religion = nicene
+					culture = greek
+					race = ROOT
+					female = no
+					trait = eunuch
+				}
```
Comment (loup99) [inline WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt]: Wouldn't culture = ROOT make more sense? Otherwise you create Spartans when they may not exist.

File: WTWSMS/events/WTWSMS_SPARTAAA.txt [hunk 4]
```diff
@@ -0,0 +1,458 @@
...
+			}
+		}
+	}
+}
+
+character_event = { # Penalty for Spartan with Craven trait
+	id = this_is_sparta.3
+	hide_window = yes
+
+	potential = {
```
Comment (loup99) [inline WTWSMS/events/WTWSMS_SPARTAAA.txt]: Needs pretriggers.

File: WTWSMS/events/WTWSMS_SPARTAAA.txt [hunk 5]
```diff
@@ -0,0 +1,458 @@
...
+# Spartans who could not fight were put to work as Sparta's bureaucrats, artists, governors, stewards, farmers, and so on to support the warriors.
+# In fact, King Agesilaos II of Sparta was a cripple and ended up being a Spartan King!
+
+	id = this_is_sparta.5
+	hide_window = yes
+
+	potential = {
+		culture = spartan
+		religion_group = pagan_group
+	}
```
Comment (loup99) [inline WTWSMS/events/WTWSMS_SPARTAAA.txt]: Pretriggers.

File: WTWSMS/events/WTWSMS_SPARTAAA.txt [hunk 6]
```diff
@@ -0,0 +1,458 @@
...
+		has_landed_title = k_sparta
+		NOT = { 
+			religion_group = pagan_group
+			religion = hellenic_pagan
+			religion = cybelian_pagan
+			religion = neoplatonist
+			religion = mithraic_pagan
+			religion = mastimani
+		}
+	}
```
Comment (loup99) [inline WTWSMS/events/WTWSMS_SPARTAAA.txt]: Pretriggers,

File: WTWSMS/events/WTWSMS_SPARTAAA.txt [hunk 7]
```diff
@@ -0,0 +1,458 @@
...
+			character_event = { this_is_sparta.6 }
+		}
+
+		clr_global_flag = THIS_IS_SPARTA
+		set_global_flag = THIS_IS_NOT_SPARTA
+	}
+}
+
+
+# I've chosen my code carefully, Persian. #
```
Comment (loup99) [inline WTWSMS/events/WTWSMS_SPARTAAA.txt]: Could you choose your namespaces and IDs equally carefully?

File: WTWSMS/history/provinces/478 - Monemvasia.txt [hunk 8]
```diff
@@ -4,13 +4,13 @@
 title = c_monemvasia
 
 # Settlements
-max_settlements = 4
+max_settlements = 6
```
Comment (loup99) [inline WTWSMS/history/provinces/478 - Monemvasia.txt]: This shouldn't become a metropolis.

File: WTWSMS/history/provinces/478 - Monemvasia.txt [hunk 9]
```diff
@@ -4,13 +4,13 @@
...
 b_monemvasia = castle
 
+b_sparta = city
```
Comment (loup99) [inline WTWSMS/history/provinces/478 - Monemvasia.txt]: This didn't exist at the time,

File: WTWSMS/history/provinces/478 - Monemvasia.txt (hunk 8 above)
Comment (TurtleShroom) [inline WTWSMS/history/provinces/478 - Monemvasia.txt]: It is a metropolis.

File: WTWSMS/history/provinces/478 - Monemvasia.txt (hunk 9 above)
Comment (TurtleShroom) [inline WTWSMS/history/provinces/478 - Monemvasia.txt]: It did exist. The city of Sparta has always existed, plus some of my code depends on it.

File: WTWSMS/events/WTWSMS_SPARTAAA.txt (hunk 7 above)
Comment (TurtleShroom) [inline WTWSMS/events/WTWSMS_SPARTAAA.txt]: No fun allowed

File: WTWSMS/common/cultures/00_cultures.txt [hunk 10]
```diff
@@ -4604,18 +4604,19 @@ byzantine = {
...
 			Hippolytos Hypatios Iakobos_Jacob Ignatios_Ignatius Innokentios Ioannes_John Ioannikios Iordanes Ioseph_Joseph Isaakios Isaias Isidoros Ioulianos_Julian Ioustinianos_Justinian Ioustinos_Justin Kaisarios_Caesar
-			Kallinikos Kallistos Komitas Konstantinos_Constantine Konstantios_Constantius Konstas_Constans Kosmas_Cosmas Kyriakos Kyrillos_Cyril Kyros Laurentios_Laurence Lazaros Leon_Leo Leontios_Leontius Loukas Makarios Manuel Markos_Marcus
-			Matthaios_Matthew Maurikios Maximos_Maximus Megistos_Magnus Meletios Methodios Michael_Michael Narses Neophytos Nikephoros Niketas Nikodemos Nikolaos_Nicholas Niphon Orestes Pamphilios Pankratios Pantoleon Paschalis
+			Kallinikos Kallistos Komitas Kleombrotos Konstantinos_Constantine Konstantios_Constantius Konstas_Constans Kosmas_Cosmas Kyriakos Kyrillos_Cyril Kyros Laurentios_Laurence Lazaros Leon Leo Leonidas Leontios_Leontius Loukas Makarios Manuel Markos_Marcus
+			Matthaios_Matthew Maurikios Maximos_Maximus Megistos_Magnus Meletios Methodios Michael Nabis Narses Neophytos Nikephoros Niketas Nikodemos Nikolaos_Nicholas Niphon Orestes Pamphilios Pankratios Pantoleon Paschalis
```
Comment (TurtleShroom) [inline WTWSMS/common/cultures/00_cultures.txt]: I thought "Micheal Micheal" was a mistake. Why would someone be named "Micheal Micheal Swag" or something like that?

File: WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt (hunk 1 above)
Comment (TurtleShroom) [inline WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt]: It does work, though.

File: WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt [hunk 11]
```diff
@@ -0,0 +1,398 @@
...
+
+			religion_group = pagan_group
+
+			NOT = { tier = EMPEROR }
+			NOT = { trait = craven }
+		}
+
+		effect = {
+			custom_tooltip = {
+				name = this_is_sparta.8
```
Comment (TurtleShroom) [inline WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt]: Says who, and why?

File: WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt [hunk 12]
```diff
@@ -0,0 +1,398 @@
...
+								religion = hellenic_pagan
+								religion = cybelian_pagan
+								religion = neoplatonist
+								religion = mithraic_pagan
+								religion = mastimani
+							}
+						}
+					}
+				}
+			}
```
Comment (TurtleShroom) [inline WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt]: Spartans were technically Greeks. Giving Hellenic Greeks a chance to become Spartan (while giving all pagan Tsakonians the chance) expands the oppurtunities for fun.

File: WTWSMS/common/scripted_triggers/WtWSMS_scripted_triggers.txt [hunk 13]
```diff
@@ -28,6 +28,7 @@ has_romance_culture_trigger = {
 		culture = helleno_thracian
 		culture = massalian
 		culture = romano_dacian
+		culture = spartan
```
Comment (TurtleShroom) [inline WTWSMS/common/scripted_triggers/WtWSMS_scripted_triggers.txt]: I agree that Spartans shouldn't be able to create Legions or restore them, but I wanted Spartans to have a chance to form Rome.

File: WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt [hunk 14]
```diff
@@ -37,26 +37,64 @@ k_sparta = {
...
+
+		mercenary = yes
+
+		title = "THREE_HUNDRED_TITLE"
+		foa = "WARCHIEF_FOA"
+
+		landless = yes
+		primary = yes
+
+		dignity = 300
```
Comment (TurtleShroom) [inline WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt]: Look at the number: do you see the reference?

File: WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt [hunk 15]
```diff
@@ -37,26 +37,64 @@ k_sparta = {
...
+							religion = mithraic_pagan
+							religion = mastimani
+						}
+					}
+				}
+			}
+
+			religion_group = pagan_group
+
+			hidden_tooltip = { always = no # Requires special Decision }
```
Comment (TurtleShroom) [inline WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt]: No. You need to either be Spartan or Tsakonian (of any pagan faith), or Greek and Hellenic.

My entry of "RELIGION = PAGAN_GROUP" is a check on Tsakonians and Spartans, while being Greek and Hellenic is more specific.

File: WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt [hunk 16]
```diff
@@ -37,26 +37,64 @@ k_sparta = {
 	capital = 478 # Sparta
 
 	culture = tsakonian
-	religion = hellenic_pagan
```
Comment (TurtleShroom) [inline WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt]: I didn't notice that. Sorry.

File: WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt [hunk 17]
```diff
@@ -0,0 +1,27 @@
...
+
+#	revoke_reason = yes # Cowardly Spartans can be relieved of their duties by their liege
+#	banishment_reason = yes # Cowardly Spartans were expelled from Sparta if they couldn't man up
+}
+
+inferior_spartan_child = { # A deformed or otherwise unworthy child was left by Spartan society to die, or roll back down the hill and be embraced as a Spartan
+	icon = 42
+
+	martial = -5 # Negates Spartan cultural bonus
+	pagan_opinion = -30
```
Comment (TurtleShroom) [inline WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt]: There's no "HELLENIC_OPINION".

File: WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt [hunk 18]
```diff
@@ -0,0 +1,398 @@
+decisions = {
+	this_is_sparta = {
+		is_high_prio = yes
```
Comment (TurtleShroom) [inline WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt]: Decisions can't have pre-triggers. Only Events can.

File: WTWSMS/common/cultures/00_cultures.txt [hunk 19]
```diff
@@ -4644,7 +4645,13 @@ byzantine = {
 		
 		founder_named_dynasties = yes
 
-		modifier = default_culture_modifier
+		allow_looting = yes
+
+		character_modifier = {
+			martial = 4
```
Comment (TurtleShroom) [inline WTWSMS/common/cultures/00_cultures.txt]: Wrong. Spartan culture was entirely militaristic. Four is completely historically accurate.

File: WTWSMS/common/traits/WTWSMS_SPARTA.txt [hunk 20]
```diff
@@ -0,0 +1,10 @@
+lame_spartan = {
+	potential = {
+		culture = spartan
+	}
+
+	martial = -6
+	diplomacy = 2
+	stewardship = 2
+	learning = 2
```
Comment (TurtleShroom) [inline WTWSMS/common/traits/WTWSMS_SPARTA.txt]: The malus is to negate the Spartan boost to Martial, minus two.

File: WTWSMS/events/WTWSMS_SPARTAAA.txt (hunk 4 above)
Comment (TurtleShroom) [inline WTWSMS/events/WTWSMS_SPARTAAA.txt]: Okay.

File: WTWSMS/events/WTWSMS_SPARTAAA.txt (hunk 5 above)
Comment (TurtleShroom) [inline WTWSMS/events/WTWSMS_SPARTAAA.txt]: Okay.

File: WTWSMS/events/WTWSMS_SPARTAAA.txt (hunk 6 above)
Comment (TurtleShroom) [inline WTWSMS/events/WTWSMS_SPARTAAA.txt]: Okay.

File: WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt [hunk 21]
```diff
@@ -0,0 +1,27 @@
+this_is_not_sparta = { # Spartan cultured character is a coward, AKA unthinkable in Spartan culture
+	icon = 90
+
+	general_opinion = -30
+	monthly_character_prestige = -1
+
+	martial = -20 # Prevents a Spartan AI ruler from choosing characters with the Craven Trait as Commanders
```
Comment (TurtleShroom) [inline WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt]: Without the Martial malus, they could still be elected as Commanders if their stats were high enough. It needs to ensure they have zero, so that there is no chance of being picked.

File: WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt (hunk 17 above)
Comment (TurtleShroom) [inline WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt]: I don't know how. I would also have to create one for every Hellenic heresy.

File: WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt (hunk 15 above)
Comment (loup99) [inline WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt]: Then add an OR block.

File: WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt (hunk 14 above)
Comment (loup99) [inline WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt]: Yes,but remember the bugs caused by adding dignity where it serves no purpose. 😉

File: WTWSMS/common/scripted_triggers/WtWSMS_scripted_triggers.txt (hunk 13 above)
Comment (loup99) [inline WTWSMS/common/scripted_triggers/WtWSMS_scripted_triggers.txt]: What? They have no connection to the unified Roman Empire.

File: WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt (hunk 18 above)
Comment (loup99) [inline WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt]: Completely wrong.

File: WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt (hunk 11 above)
Comment (loup99) [inline WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt]: Because it completely messes up localisation and creates conflicts for no purpose.

File: WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt (hunk 1 above)
Comment (loup99) [inline WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt]: It should get removed upon the next succession though.

File: WTWSMS/common/cultures/00_cultures.txt (hunk 10 above)
Comment (loup99) [inline WTWSMS/common/cultures/00_cultures.txt]: No, that is because you haven't understood how name lists work then.

File: WTWSMS/common/cultures/00_cultures.txt (hunk 19 above)
Comment (loup99) [inline WTWSMS/common/cultures/00_cultures.txt]: Yes, so were others, who get lesser bonuses.

File: WTWSMS/events/WTWSMS_SPARTAAA.txt (hunk 7 above)
Comment (loup99) [inline WTWSMS/events/WTWSMS_SPARTAAA.txt]: When they are confusing they don't work. One event is namescape.number with namescape.number.desc, and then namescape.number.letter for the options.

File: WTWSMS/history/provinces/478 - Monemvasia.txt (hunk 8 above)
Comment (loup99) [inline WTWSMS/history/provinces/478 - Monemvasia.txt]: Huh? It shouldn't be one, which is why this change doesn't fit.

File: WTWSMS/history/provinces/478 - Monemvasia.txt (hunk 9 above)
Comment (loup99) [inline WTWSMS/history/provinces/478 - Monemvasia.txt]: You can create it through effects if that is the problem.

File: WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt (hunk 21 above)
Comment (loup99) [inline WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt]: They can still be picked at 0.

File: WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt (hunk 17 above)
Comment (loup99) [inline WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt]: Look at how it is done.

> I would also have to create one for every Hellenic heresy.


Yes? That is better than it being broken.

File: WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt (hunk 21 above)
Comment (TurtleShroom) [inline WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt]: They won't be picked if anyone is better than them. How can we prevent people with this Modifier from being picked at all?

File: WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt (hunk 17 above)
Comment (TurtleShroom) [inline WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt]: Dude, I don't know how to do that. Feel free to add it yourself.

File: WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt (hunk 15 above)
Comment (TurtleShroom) [inline WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt]: No, if I did that, then it would be Tsakonian OR Greek and Hellenic OR in the Pagan Group, allowing all Pagans to reform Sparta!

File: WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt (hunk 14 above)
Comment (TurtleShroom) [inline WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt]: The bugs only apply when the Dignity is too high.

File: WTWSMS/common/scripted_triggers/WtWSMS_scripted_triggers.txt (hunk 13 above)
Comment (TurtleShroom) [inline WTWSMS/common/scripted_triggers/WtWSMS_scripted_triggers.txt]: If they are revived in this time period, they should be able to reform their empire of old, and in this case, usurp the Roman Empire. All Greeks have heritage from Rome, including the Tsakonians who can become Spartan.

File: WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt (hunk 21 above)
Comment (loup99) [inline WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt]: Probably through playing with AI modifiers.

File: WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt (hunk 15 above)
Comment (loup99) [inline WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt]: At the moment no one can do it, because you can't be both Greek and Hellenic AND Spartan or Tsakonian as you ask.

File: WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt (hunk 11 above)
Comment (TurtleShroom) [inline WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt]: It does not mess up localization, and there are no conflicts.

In addition, "THIS IS SPARTA!!" is used as an Option in Events! I simply reused that same Event Option as a tooltip elsewhere. Otherwise, I would need TWO "THIS IS SPARTA!!" keys in Localization, which is stupid.

File: WTWSMS/common/cultures/00_cultures.txt (hunk 10 above)
Comment (TurtleShroom) [inline WTWSMS/common/cultures/00_cultures.txt]: What do you mean by "name list links". The guy's name would be "Micheal Micheal". Do you have a source on a Spartan being named "Micheal Micheal"?

File: WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt (hunk 14 above)
Comment (loup99) [inline WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt]: primary = yes defeats the purpose of dignity.

File: WTWSMS/common/cultures/00_cultures.txt (hunk 19 above)
Comment (TurtleShroom) [inline WTWSMS/common/cultures/00_cultures.txt]: Spartans are special, though, and other cultures do not normally get Martial boosts, even if they could.

File: WTWSMS/common/scripted_triggers/WtWSMS_scripted_triggers.txt (hunk 13 above)
Comment (loup99) [inline WTWSMS/common/scripted_triggers/WtWSMS_scripted_triggers.txt]: No, not at all. The Roman Empire can't be usurped and it has no connection to Ancient Greeks.

File: WTWSMS/common/traits/WTWSMS_SPARTA.txt (hunk 20 above)
Comment (TurtleShroom) [inline WTWSMS/common/traits/WTWSMS_SPARTA.txt]: They reflect Spartan society's direction of the cripples to other disciplines, which they mastered.

File: WTWSMS/history/provinces/478 - Monemvasia.txt (hunk 8 above)
Comment (TurtleShroom) [inline WTWSMS/history/provinces/478 - Monemvasia.txt]: That land is very fertile and has huge potential for increase in population. It does not _start_ with those titles full.

File: WTWSMS/history/provinces/478 - Monemvasia.txt (hunk 9 above)
Comment (TurtleShroom) [inline WTWSMS/history/provinces/478 - Monemvasia.txt]: Spawning a city out of nothing makes no sense, when the city is already there.

File: WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt (hunk 11 above)
Comment (loup99) [inline WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt]: This is a custom tooltip. We don't need any tooltip which reads "THIS IS SPARTA".

File: WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt (hunk 21 above)
Comment (TurtleShroom) [inline WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt]: Could we go into the Commander Minor Title and prevent people with this modifier from getting the office?

File: WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt (hunk 15 above)
Comment (TurtleShroom) [inline WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt]: What do you mean? It doesn't look like that to me. I'll re-check, though.

File: WTWSMS/common/scripted_triggers/WtWSMS_scripted_triggers.txt (hunk 13 above)
Comment (TurtleShroom) [inline WTWSMS/common/scripted_triggers/WtWSMS_scripted_triggers.txt]: Okay.

File: WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt (hunk 11 above)
Comment (TurtleShroom) [inline WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt]: The custom tooltip is also an Event Option. We do need a tooltip which reads "THIS IS SPARTA", because I say we need one and it makes sense in the code. People that play this Mod will understand both the reference and the humor, because they actually have a sense of humor.

File: WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt (hunk 21 above)
Comment (loup99) [inline WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt]: Yes.

File: WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt (hunk 17 above)
Comment (loup99) [inline WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt]: Then I will add it to the list of things to fix on this pull-request.

File: WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt (hunk 15 above)
Comment (loup99) [inline WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt]: I mean that it lacks an OR.

File: WTWSMS/common/cultures/00_cultures.txt (hunk 10 above)
Comment (loup99) [inline WTWSMS/common/cultures/00_cultures.txt]: Look up things before coming with uniformed comments. There is no name "Micheal Micheal", and no it wouldn't, that is just because you don't know how to mod cultures.

"Variations of the same name can be defined by joining them with an underscore: Johan_John. This allows regnal numbering to be consistent if culture changes, so Juan III won't be succeeded by Ivan I. Double names can also be defined by enclosing them in double quotes. E.g. "Sarah Ann". These will be treated as a single name."

File: WTWSMS/common/cultures/00_cultures.txt (hunk 19 above)
Comment (loup99) [inline WTWSMS/common/cultures/00_cultures.txt]: You aren't addressing my point at all: it is unbalanced.

File: WTWSMS/history/provinces/478 - Monemvasia.txt (hunk 8 above)
Comment (loup99) [inline WTWSMS/history/provinces/478 - Monemvasia.txt]: ❔ I think you don't understand how prosperity mechanics works.

File: WTWSMS/common/cultures/00_cultures.txt (hunk 10 above)
Comment (TurtleShroom) [inline WTWSMS/common/cultures/00_cultures.txt]: Oh! Okay then.

File: WTWSMS/common/cultures/00_cultures.txt (hunk 19 above)
Comment (TurtleShroom) [inline WTWSMS/common/cultures/00_cultures.txt]: It is not unbalanced. Becoming Spartan is a long and difficult process that should reap huge rewards. Like you suggested, you cannot instantly become Spartan once you restore Sparta.

File: WTWSMS/history/provinces/478 - Monemvasia.txt (hunk 8 above)
Comment (TurtleShroom) [inline WTWSMS/history/provinces/478 - Monemvasia.txt]: I wasn't talking about prosperity. I was talking about empty slots for potential holdings.

Comment (loup99) [inline WTWSMS/history/provinces/478 - Monemvasia.txt]: Prosperity is a system which adds holdings. This place was neither fertile nor populous historically in the timeframe. But if you change that you can get 6 holdings.

File: WTWSMS/common/cultures/00_cultures.txt (hunk 19 above)
Comment (loup99) [inline WTWSMS/common/cultures/00_cultures.txt]: Yes, not all with one modifier.

File: WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt (hunk 21 above)
Comment (TurtleShroom) [inline WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt]: Okay, I'll do that.

File: WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt (hunk 15 above)
Comment (TurtleShroom) [inline WTWSMS/common/landed_titles/WtWSMS_erik_titles.txt]: Ah. Okay,

File: WTWSMS/history/provinces/478 - Monemvasia.txt (hunk 8 above)
Comment (TurtleShroom) [inline WTWSMS/history/provinces/478 - Monemvasia.txt]: Prosperity doesn't add new holding slots. Changing that line allows for six empty holding slots, as opposed to three. It was intended to prevent the City of Sparta from not being able to be added, but I realized that you can force a new holding slot, so I'll change that now.

File: WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt (hunk 17 above)
Comment (TurtleShroom) [inline WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt]: Okay. Make sure you add a "SPARTAN_OPINION" for the Spartan culture as well, plus an "ANTINOMIANISM_OPINION". In a future PR, I'm going to add opinions so that Antinomians are not effected by the maluses to adultery and the like, etc. .

File: WTWSMS/history/provinces/478 - Monemvasia.txt (hunk 8 above)
Comment (IhateTrains) [inline WTWSMS/history/provinces/478 - Monemvasia.txt]: Prosperity DOES add new holding slots.

File: WTWSMS/common/cultures/00_cultures.txt (hunk 19 above)
Comment (TurtleShroom) [inline WTWSMS/common/cultures/00_cultures.txt]: The Character Modifiers applies to those of the Spartan culture, and _only_ to the Spartan culture. There is supposed to be a long and arduos trial to become Spartan, so the Character Modifiers are only added after the journey is finished.

**Comment by TurtleShroom** [comment]
Okay, I've answered the questions and made the changes. Please look through it again, so that we may Commit it.

File: WTWSMS/common/cultures/00_cultures.txt (hunk 19 above)
Comment (loup99) [inline WTWSMS/common/cultures/00_cultures.txt]: Why would it apply to other cultures too? You haven't understood culture modding: you can't add these bonuses to a group, it is per culture only.

Comment (TurtleShroom) [inline WTWSMS/common/cultures/00_cultures.txt]: It doesn't! I ONLY wrote that for Spartan culture, not all the others!

Comment (loup99) [inline WTWSMS/common/cultures/00_cultures.txt]: It can't anyway, it is technically impossible to do it in this manner, so I don't know why you suggested it. Regardless of your confusion, which I advice you to resolve through reading the wiki, the character modifier itself remains unbalanced. And no, you don't resolve a lack of balance through distributing modifiers to other cultures.

Comment (TurtleShroom) [inline WTWSMS/common/cultures/00_cultures.txt]: There is no confusion. Cultures can have their own modifiers that apply to the culture. I didn't want to distribute modifiers to other cultures. The Spartan culture gets a significant Martial boost because that is historical.

I do not appreciate you belittling me by telling me that I haven't read the Wiki or looked into the code.

Given that you issued a thumbs down to my post, you clearly didn't read the code, because I ONLY gave that modifer to the Spartan culture, not the Byzantine Group.

untitled

Comment (loup99) [inline WTWSMS/common/cultures/00_cultures.txt]: No but seriously now. First of all, never use Notepad for code editing. Secondly I will repeat my prior post:

> Why would it apply to other cultures too? 

As I said, and I will rephrase to make it very clear, the question is not about how you get the modifier or who gets it (only Spartans should as I said, and if you agree on that there is no contention), it is concerning the fact that the +4 martial is a too significant modifier for what it is meant to represent.

File: WTWSMS/decisions/WtWSMS_form_titles_decisions.txt [hunk 22]
```diff
@@ -390,7 +390,7 @@ decisions = {
 }
 
 decisions = {
-	this_is_sparta = {
+	this_is_sparta = { # Edits and commentary courtesy of Loup
```
Comment (loup99) [inline WTWSMS/decisions/WtWSMS_form_titles_decisions.txt]: Yeah, although you didn't implement all I said and added new things I disagree on.

**Comment by loup99** [comment]
Closing this pull-request as the creator has left the development team and is no longer a part of the project.
